release_notes.md

```diff
@@ -12,5 +12,5 @@
 - Upgrade PowerShell language worker 7.0 to 4.0.2733 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2733)
```

Comment:
Please clear release notes that are present in [`4.19.0`.](https://github.com/Azure/azure-functions-host/releases/tag/v4.19.0).

Reply:
updated as per the comments